src/screens/Asset/ReceiveModal.js

```diff
@@ -138,79 +81,53 @@ const ReceiveModal = ({
 
   const { profileImage, lastUpdateTime = 0, username = '' } = user;
   const ensName = getAccountEnsName(activeAccount);
-  const needsSmallButtons = showBuyTokensButton && SCREEN_WIDTH < 300;
   const profileImageURI = profileImage ? `${profileImage}?t=${lastUpdateTime}` : null;
 
   return (
     <SlideModal
       onModalHide={onModalHide}
       noPadding
       noClose
-      headerLeftItems={showErc20Note ? [{
-        custom: (
-          <LabelBadge
-            label={t('label.erc20TokensOnly')}
-            labelStyle={{ fontSize: fontSizes.tiny }}
-            primary
-          />
-        ),
-      }] : undefined}
+      // TODO : The label itself will be revisited later on the direction of the product team
+      // headerLeftItems={showErc20Note ? [{
+      //   custom: (
+      //     <LabelBadge
+      //       label={t('label.erc20TokensOnly')}
+      //       labelStyle={{ fontSize: fontSizes.tiny }}
+      //       primary
+      //     />
+      //   ),
+      // }] : undefined}
       centerFloatingItem={
         <ImageWrapper style={{ position: 'absolute', marginTop: -24 }}>
-          <ProfileImage
-            uri={profileImageURI}
-            userName={username}
-            diameter={48}
-          />
+          <ProfileImage uri={profileImageURI} userName={username} diameter={48} />
         </ImageWrapper>
       }
     >
       <ContentWrapper forceInset={{ top: 'never', bottom: 'always' }}>
-        <WarningBanner rounded small />
-        {!!ensName && (
-          <InfoView>
-            <BaseText
-              big
-              onPress={() => handleCopyToClipboard(ensName, true)}
-              center
-            >
+        <InfoView>
+          {!!ensName && (
+            <TextWithCopy textToCopy={ensName} iconColor="#007aff">
```

Comment:
```suggestion
            <TextWithCopy textToCopy={ensName} iconColor="#007aff">
```

Is the icon colour referenced from any common colours?

Reply:
Done.